style.css

```diff
@@ -0,0 +1,444 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0; 
+}
+html{
+    background-color:#6EA4BF;
+    font-family: Arial, Helvetica, sans-serif;
+    /* height: 100%; */
+}
+
+body{
+    /* display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%; */
+    padding-bottom:100px;
+}
+
+.profile_pic_center{
+    text-align:center;
+}
+
+img{
+    max-width: 100%;
+    /* width: 200px; */
+    height:300px;
+    margin-top: 20px;
+    /* margin-left: 20%; */
+}
+
+.intro h4{
+    margin-bottom:8px;
+    padding-left:5px;
+}
+
+.intro h2{
+    font-size: 2rem;
+    font-style: italic;
+    padding-left:5px;
+}
+
+.intro h3{
+    margin-bottom:10px;
+    padding-left:5px;
+}
+
+.about-me{
+    padding-left:5px;
+}
+
+.main{
+    flex: 1 0 auto;
+}
+
+
+
+.header {
+    display:flex;
+    background-color:#1C3144;
+    border: 5px #6EA4BF solid;
+}
+
+.header__title{
+    color:#FBFEF9; 
+    padding: 2px;
+    font-size: 25px;
+    margin-left: 5px;
+    margin-right: 12px;
+    margin-top: 5px;
+}
+
+.header__menu-item{
+    display: inline-block;
+    list-style:none;
+    text-decoration:none;
+    margin-left:10px;
+    padding-left:4px;
+}
+
+
+.header__menu-item a{
+    text-decoration: none; 
+    color:#FBFEF9;
+    font-size: 12px;
+    margin-left:9.5px;
+}
+
+.navigate{
+    display:flex;
+}
+
+.fa{
```

Comment:
It looks like you're overriding the Font Awesome styles here. When using classes provided by external CSS, I will usually add my own classes to the HTML tags and then override targeting my own classes. That isn't necessary, but can serve to make sure you don't accidentally override other Font Awesome usages elsewhere in your site.